src/main/java/xades4j/verification/SignatureSpecificVerificationOptions.java

```diff
@@ -61,6 +62,25 @@ String getBaseUri()
         return this.baseUriForRelativeReferences;
     }
 
+    public boolean checkKeyUsage()
+    {
+        return checkKeyUsage;
+    }
+
+    /**
+     * Configures whether to check that the keyUsage of the signer certificate
+     * allows use for signing. If enabled (the default) signature validation will
+     * fail if the keyUsage of the certificate does not allow signing.
+     *
+     * @param enabled {@code true} to enable the check, {@code false to disable}
```

Comment:
"to disable" should be outside the code tag

Reply:
How did I do that again? Sorry, fixed in 1ced0a8e1bafc6da38c54010d410cdb7271acfc8. :sweat_smile: